.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: emacs-lisp
+sudo: no
+cache:
+  directories:
+    - $TRAVIS_BUILD_DIR/.cask
+env:
+  - EVM_EMACS=emacs-24.3-travis
```

Comment:
Is there any point to test on every minor version? Maybe only major versions is enough?

Reply:
Probably not. [Here](https://github.com/bbatsov/projectile/blob/master/.github/workflows/test.yml#L17)'s an example GitHub Actions config that uses the earliest and latest for each stable branch. That seems perfectly sane to me. I plan to replace Travis CI config today or some time soon if I don't have time today.